docs/coordinates/representations.rst

```diff
@@ -31,6 +31,18 @@ The built-in representation classes are:
   cylindrical polar coordinates, represented by a cylindrical radius
   (``rho``), azimuthal angle (``phi``), and height (``z``).
 
+
+Astropy also offers a `~astropy.coordinates.BaseGeodeticRepresentation` useful to
+create specific representations on spheroidal bodies (see
```

Comment:
I think this paragraph is very clear! Should we mention the existing earth-based ones? E.g.,
```

Astropy also offers a `~astropy.coordinates.BaseGeodeticRepresentation` useful to
create specific `repreentations` on spheroidal bodies. This is used internally for the
standard Earth ellipsoids used in `~astropy.coordinates.EarthLocation` 
(`~astropy.coordinates.WGS84GeodeticRepresentation`,
`~astropy.coordinates.WGS72GeodeticRepresentation`, and
`~astropy.coordinates.GRS80GeodeticRepresentation`), but 
can also be customized; see :ref:`astropy-coordinates-create-geodetic`.
All these are coordinates on ...
```

Reply:
I like it! I'm adding it right now.